Update pathology tumor detection readme (#584)
Project-MONAI/model-zoo: models/pathology_tumor_detection/docs/README.md

A probability number of the input patch being tumor or normal.

### Memory Consumption Warning

> If you face memory issues in traning, you can lower the `batch_size` of the validation dataloader in the configurations to reduce the System RAM requirements.

Reviewer: ```suggestion
If you face memory issues in traning, you can lower the `batch_size` or `patch_size` in the configurations to reduce the System RAM requirements.
```

Not only validation? Maybe the batch size of training could also be reduced？

#### Override the `train` config to execute multi-GPU training

```
torchrun --nnodes=1 --nproc_per_node=2 -m monai.bundle run --config_file "['configs/train.json','configs/multi_gpu_train.json']"
```

> Please note that the distributed training-related options depend on the actual running environment; thus, users may need to remove `--standalone`, modify `--nnodes`, or do some other necessary changes according to the machine used. For more details, please refer to [pytorch's official tutorial](https://pytorch.org/tutorials/intermediate/ddp_tutorial.html).

Reviewer: Looks we can remove "to remove `--standalone`" here?

Author: thanks, I removed this change.
